commands/src/main/java/org/wildfly/extras/creaper/commands/ra/TransactionType.java

```diff
@@ -0,0 +1,16 @@
+package org.wildfly.extras.creaper.commands.ra;
+
+
+public enum TransactionType {
+    XA("XATransaction"),
+    NONE("NoTransaction");
+    String value;
```

Comment:
Should be `private`. And there should be an empty line between the enum constants and fields.